kafka-env/postgres/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM postgres:13.0-alpine
+FROM debezium/postgres:13
```

Comment:
ohh mate, why so complex?
you may use postgress in docker-compose directly and mount initdb volume on it.